src/zoom-provider.ts

```diff
@@ -119,27 +109,26 @@ function TangocardProvider(this: any, options: TangocardProviderOptions) {
   })
 
 
-
   seneca.prepare(async function(this: any) {
     let res =
-      await this.post('sys:provider,get:keymap,provider:tangocard')
+      await this.post('sys:provider,get:keymap,provider:zoom')
 
     if (!res.ok) {
       throw this.fail('keymap')
     }
 
-    let src = res.keymap.name.value + ':' + res.keymap.key.value
-    let auth = Buffer.from(src).toString('base64')
-
+    const src = res.keymap.client_id.value + ':' + res.keymap.client_secret.value
+    const auth = Buffer.from(src).toString('base64')
     this.shared.headers = {
```

Comment:
is this used?

Reply:
@rjrodger It is. 

Because we need to use the account credentials to request the access token.
My idea here was to make a post request to get the access token.
Thus, adding the auth (containing the account credentials) to the headers object before we make the request.

And then, make the request here:
https://github.com/lmscunha/seneca-zoom-provider/blob/9ca812d56803d23636a704a62e8ed6d7bb5826c6/src/zoom-provider.ts#L129

Please click here for more details:
https://developers.zoom.us/docs/internal-apps/s2s-oauth/#use-account-credentials-to-get-an-access-token